api/config.py

```diff
@@ -262,11 +280,14 @@ def get_config():
     return __config
 
 def get_public_config():
+    auth = copy.deepcopy(__config.get('auth'))
+    for values in auth.itervalues():
+        values.pop('client_secret', None)
```

Comment:
This is blacklisting when the below return whitelists. Thoughts on making this section a whitelist also for safety?